<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataImportTool</class>
 <widget class="QWidget" name="DataImportTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>465</width>
    <height>94</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Excel数据导入工具</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="user_label">
       <property name="text">
        <string>选择Excel文件：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="user_path_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="user_path_button">
       <property name="text">
        <string>选择</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>38</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="import_button">
       <property name="text">
        <string>导入到数据库</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
